visualization/app/codeCharta/ui/rangeSlider/rangeSlider.component.ts

```diff
@@ -59,6 +63,21 @@ export class RangeSliderController implements SettingsServiceSubscriber {
         this.settingsService.applySettings();
     }
 
+    private updateSliderColors() {
+        const slider = $("range-slider-component .rzslider");
+        const leftSection = slider.find(".rz-bar-wrapper:nth-child(3) .rz-bar");
+        const middleSection = slider.find(".rz-selection");
+        const rightSection = slider.find(".rz-right-out-selection .rz-bar");
+        const fromPercentage = 100 / this.maxMetricValue * this.settingsService.settings.neutralColorRange.from;
+
+        const leftColor = this.settingsService.settings.neutralColorRange.flipped ? MapColors.negative : MapColors.positive;
+        const rightColor = this.settingsService.settings.neutralColorRange.flipped ? MapColors.positive : MapColors.negative;
+
+        leftSection.css("cssText", "background: #" + leftColor.toString(16) + " !important; width: " + fromPercentage + "%;");
```

Comment:
Does anybody know how to test, that the colors are set inside the html `rzslider`?

Reply:
There are several options for this: 
1) assume that .css(...) works correctly and the library introducing this method is tested. Then you could just spy on this method and check if the correct parameters are given. 
2)[You could use $compile service from angular js](https://stackoverflow.com/questions/24438069/css-testing-in-angularjs). Note: We have no example for this in CC. This would be something new. 
3) UI test. Propably overkill for this test and also does not measure code coverage. 

I personally think number 1 is good enough.

Reply:
Okey thanks for your suggestions! 👍

Reply:
Everything else looks fine. You can merge it after testing